### DISCUSSION TITLE
How to add more space to the root partition (/)

### DISCUSSION THREAD

**Discussion by RuizSenior** [prasanthrangan/hyprdots#1416] [Q&A]
### Description

I know this is not an arch-focused github, but the community here is very active. I am quite new to arch and I followed tutorials for its installation and in "Disk configuration" select "Use a best-effort default partition layout". But by selecting that option only assigned 20 GB to root (/). Normally when I used Ubuntu I assigned 100 GB to the root and the rest to /home, how can I add space from /home to the root?

### Screenshots

![image](https://github.com/prasanthrangan/hyprdots/assets/91767821/149b99e7-d17d-4896-8b78-ee9451e5b8b6)

**Comment by RuizSenior** [comment]
Btw, Is it normal that hyprdots take up so much space?  I have installed very few packages.

**Comment by kRHYME7** [comment]
H! 
> I know this is not an arch-focused github, but the community here is very active

Lol, I find this abusing. Please join the discord server, so other users can help you. Here's the link https://discord.com/invite/qWehcFJxPa .  
https://www.youtube.com/results?search_query=arch+install  youtube will probably help you properly install arch.

>  "Disk configuration" select "Use a best-effort default partition layout"

 20Gb is not enough if you want to have a decent experience
Also, you can follow some YouTube videos covering installation of ARCH.
https://www.youtube.com/results?search_query=arch+install 

> Btw, Is it normal that hyprdots take up so much space? I have installed very few packages.

No, this is lightweight. It might be you installed some flatpak apps too, I guess. 

I will move this to discussions panel where this topic is suited.

**Comment by mahaveergurjar** [comment]
Run live any Linux distribution that is support gparted. After run live Bootable iso of Linux then open gparted and reduce the size of home partition and the unlocated space size that  reduce from home add in root partition

**Reply by Enigma1309** [reply]
that isn't possible without data loss

**Comment by mahaveergurjar** [comment]
It is possible i already try this

**Reply by mahaveergurjar** [reply]
And no data lose

**Comment by mahaveergurjar** [comment]
This is work only when you don't use encryption

**Comment by mahaveergurjar** [comment]
Wait i will do this and then show you

**Comment by mahaveergurjar** [comment]
See this before increase the size of home partition 
![240504_21h10m33s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/fd03db35-9c78-4b44-8974-1d695ca15acf)

Steps to increase home partition size 
![IMG_20240504_211400](https://github.com/prasanthrangan/hyprdots/assets/57872034/6ce92abd-2157-4520-b52f-697433a6cb20)
![IMG_20240504_211501](https://github.com/prasanthrangan/hyprdots/assets/57872034/2eaac4fb-9bce-449b-8387-1065ffa60e71)
![IMG_20240504_211539](https://github.com/prasanthrangan/hyprdots/assets/57872034/90653f6b-2130-4475-8af8-24312f7fae08)
![IMG_20240504_211547](https://github.com/prasanthrangan/hyprdots/assets/57872034/d37dbe92-29f5-441c-95df-76fb7b6ba2c7)
![IMG_20240504_212527](https://github.com/prasanthrangan/hyprdots/assets/57872034/ac8834cd-a5d9-4651-b25b-128de3511748)

After increase the size of home partition
![240504_21h27m16s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/0fc93666-22f8-4fe4-9e90-fd6388658020)

**Comment by mahaveergurjar** [comment]
Warning ⚠️ Don't apply this method on boot partition and encrypted drive

**Comment by Enigma1309** [comment]
@mahaveergurjar what you did was to resize the home partition and reduce root (this is possible)
but you can't do the other way around (reduce home and increase root) since the os writes data to the hdd/ssd at the partition start sector. 
ie you can't reduce home dir and increase root without actually destroying home but it can be done other way round...
(there's an exception that you may create a home partition before root and the above said things will be changed respectively)

**Comment by mahaveergurjar** [comment]
This is also possible I try this also that you say that os write data in starting that is right this problem only on legacy bios system new system are used efi system and in this possibile to change the location because os or boot from boot partition so the boot partition are not you change the size

**Comment by mahaveergurjar** [comment]
And on the above of the user picture you see the root partition are above on the home partition so reduce home from starting and merge in root partition of the end didn't not effect the staring sectors of of root partition

**Comment by Enigma1309** [comment]
okay i guess i maybe wrong but in my case it don't show resizing from start but thank @mahaveergurjar for info i'll try this out in future

**Comment by mahaveergurjar** [comment]
Ok when i am free then resize and show

**Reply by Enigma1309** [reply]
i just tried iso live boot it and it was showing an option to resize which means i can do it..
i tried it and it started copying the data (180 gb X0 ) it was showing a taking long time so i cancelled and moved on (fortunately my data is safe)  
i guess i was partially wrong

**Reply by mahaveergurjar** [reply]
are you reduce the size ??

**Reply by mahaveergurjar** [reply]
yes don't do this if opertion are running

**Reply by mahaveergurjar** [reply]
always try to add the  unlocated partitiion last  in root partition

**Comment by mahaveergurjar** [comment]
![IMG_20240504_225922](https://github.com/prasanthrangan/hyprdots/assets/57872034/84eb64ac-74cb-4279-9eef-e07c14c7db81)
![IMG_20240504_230000](https://github.com/prasanthrangan/hyprdots/assets/57872034/4d7e4b0e-ff18-465c-b060-4ea191a21fa3)
![IMG_20240504_232249](https://github.com/prasanthrangan/hyprdots/assets/57872034/81660f0b-dba8-43b8-9e67-e9a41d040826)
![240504_23h26m00s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/99c51bc5-f14f-491d-a35a-c616a0198e1c)

**Comment by mahaveergurjar** [comment]
you can see in image one that I reduced the partition from starting in the home and picture two you see the unallocated partition between root and home that I reduced from starting so I and this unallocated space merge  at the end of the root partition so the starting sector is also not affected in the root partition

**Reply by Enigma1309** [reply]
yeah you're correct :)
thanks for letting me know this

**Reply by mahaveergurjar** [reply]
Is your root partition increased ??

**Reply by Enigma1309** [reply]
i didn't increased it cause it was showing it will take a lot of time, but yeah i can confirm that this method works
